### PR TITLE
replace Plug.Logger with Plug.Telemetry in routing guide

### PR DESCRIPTION
**Pull request by ignatiusreza**
this seems like a leftover from 4d61eacd83d61899a895167715d8754c5c8187be ?

### CODE REVIEW

File: guides/routing.md [hunk 1]
```diff
@@ -691,7 +691,7 @@ Endpoints organize all the plugs common to every request, and apply them before
 
 - [Plug.RequestId](https://hexdocs.pm/plug/Plug.RequestId.html) - generates a unique request id for each request.
 
-- [Plug.Logger](https://hexdocs.pm/plug/Plug.Logger.html) - logs incoming requests
+- [Plug.Telemetry](https://hexdocs.pm/plug/Plug.Telemetry.html) - adds instrumentation points so Phoenix can log the srequest path, status code and request time by default.
```
Comment (mpranjic) [inline guides/routing.md]: "srequest" should be "request"

Comment (ignatiusreza) [reply guides/routing.md]: Woops, sorry about that.. fixed!

Should I squash the changes?

**Comment by ignatiusreza** [comment]
test failure doesn't seems to be related.. and master branch seems to be intermittently failing?

**Comment by jayjun** [comment]
@ignatiusreza I fixed the cause of failing test in #3497.

**Comment by ignatiusreza** [comment]
wow! thanks @jayjun! i'll rebase this branch once the fix is merged to master :+1:

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**